steps/perfecto-upload/0.0.2/step.yml

```diff
@@ -0,0 +1,66 @@
+title: Perfecto Upload
+summary: |
+  Upload APK/IPA files to Perfecto cloud repository
+description: |
+  Upload your build files to Perfecto for remote device testing. Upload your IPA/APK file to Perfecto repository
+website: https://github.com/PerfectoCode/Bitrise_Perfecto_FileUpload
+source_code_url: https://github.com/PerfectoCode/Bitrise_Perfecto_FileUpload
+support_url: https://github.com/PerfectoCode/Bitrise_Perfecto_FileUpload/issues
+published_at: 2020-06-18T21:00:14.078053+05:30
+source:
+  git: https://github.com/PerfectoCode/Bitrise_Perfecto_FileUpload.git
+  commit: 3a8cdb6ae39caa3b866d30266a02094f1b35336f
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
+- android
+- xamarin
+- react-native
+- cordova
+- ionic
+- flutter
+type_tags:
+- deploy
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: curl
+  apt_get:
+  - name: curl
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- opts:
+    description: |
+      This usually gets generated by the the "Gradle Runner" or "XCode Archive & Export" step
+    is_required: true
+    summary: The app file you want to upload to Perfecto, usually $BITRISE\_APK\_PATH
+      or $BITRISE\_IPA\_PATH
+    title: A Bitrise generated APK or IPA path
+  upload_path: $BITRISE_APK_PATH
+- opts:
+    description: |
+      The security token you use to log into your Perfecto account
+    is_required: true
+    summary: Your perfecto access key.
+    title: Perfecto access key
+  perfecto_accesskey: $PERFECTO_ACCESS_KEY
```

Comment:
Default values should not use such env. variables.
They are only permitted if normally available in environment (eg. exported by anther step, bitrise CLI or bitrise.io).